components/backup/src/writer.rs

```diff
@@ -190,6 +199,76 @@ impl BackupWriter {
     }
 }
 
+/// A writer writes Raw kv into SST files.
+pub struct BackupRawKVWriter {
+    name: String,
+    cf: String,
+    write: Writer,
+    limiter: Option<Arc<RocksIOLimiter>>,
+}
+
+impl BackupRawKVWriter {
+    /// Create a new BackupRawKVWriter.
+    pub fn new(
+        db: Arc<DB>,
+        name: &str,
+        cf: &str,
+        limiter: Option<Arc<RocksIOLimiter>>,
+    ) -> Result<BackupRawKVWriter> {
+        let write = RocksSstWriterBuilder::new()
+            .set_in_memory(true)
+            .set_cf(super::endpoint::rawkv_cf(cf)?)
+            .set_db(RocksEngine::from_ref(&db))
+            .build(name)?;
+        Ok(BackupRawKVWriter {
+            name: name.to_owned(),
+            cf: cf.to_owned(),
+            write: Writer::new(write),
+            limiter,
+        })
+    }
+
+    /// Write Kv_pair to buffered SST files.
+    pub fn write(&mut self, kv_pairs: &[Result<KvPair>], need_checksum: bool) -> Result<()> {
+        for kv_pair in kv_pairs {
+            let (k, v) = match kv_pair {
+                Ok(s) => s,
+                Err(e) => {
```

Comment:
It seems we can not reach this error if we [push OK into kv_pairs ](https://github.com/tikv/tikv/pull/6308/files#diff-53831868d19a8c3495a300c681e252a8R227)

Reply:
This code is written with reference to [this](https://github.com/tikv/tikv/blob/ab5d1d6233541cb4162d45f70a82a898a14c984e/src/storage/mod.rs#L1162)